test.js

```diff
@@ -134,6 +134,50 @@ test('execa.shellSync() includes stdout and stderr in errors for improved debugg
 	t.throws(() => execa.shellSync('node fixtures/error-message.js'), {message: STDERR_STDOUT_REGEXP, code: 1});
 });
 
+test.cb('execa() with .kill after it with SIGKILL should kill cleanly', t => {
```

Comment:
Tests should use the same style as the other tests:
  - `test.cb()` + `test.end()` should use `test()` + `async`/`await` instead. You can promisify `setTimeout` with `util.promisify(setTimeout)`
  - `proc` should be named `subprocess` as the other tests.

Reply:
Done: bb5b280c806e049600d3de4db876a7d5d6a17888